### ISSUE TITLE
Deprecate the Farbtastic color picker in core

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#5598]
The code lives in https://github.com/backdrop/backdrop/tree/1.x/core/misc/farbtastic in Backdrop.

[The project](https://github.com/mattfarina/farbtastic) has not been updated since 2017 (which makes it a potential security risk), and it has been archived for some time now. I was not able to find a maintained fork of it

Our Drupal brethren seem to be looking to remove/replace that library too. Here are some relevant issues from Drupal-land:
- [Drupal 10 JavaScript dependency plan](https://www.drupal.org/project/drupal/issues/3238507)
- [Move the Color module to a contributed project when Bartik is deprecated](https://www.drupal.org/project/ideas/issues/2808151)
- [Implement color changing theme settings for Olivero](https://www.drupal.org/project/drupal/issues/3257274)
- https://www.drupal.org/project/color

Here are some articles around the net that compare color-picking solutions:
- https://www.dunebook.com/12-best-js-color-picker-libraries (seems to be from 2019)
- https://ourcodeworld.com/articles/read/81/top-5-best-javascript-and-jquery-color-picker-plugins (2020)
- https://blog.logrocket.com/color-picker-libraries-to-use-in-2021 (2021 as suggested by the article title)

This one comes up a lot (but is in React): https://github.com/casesandberg/react-color / https://casesandberg.github.io/react-color

**Comment by indigoxela** [comment]
It should be noted that core doesn't use that library, anyway. But as it's a part of the API, it might have to stay for a while.

Actually Backdrop uses the native color input, which is widely supported nowadays.

So the long term task here is probably not to search yet another library, but to just trow out farbtastic. - Probably after some time of deprecation.

**Comment by BWPanda** [comment]
Yeah, the only place in core that seems to use this is Simpletest: https://github.com/backdrop/backdrop/search?q=farbtastic

Let's repurpose this issue for deprecating Farbtastic.
